SECURITY.md

```diff
@@ -12,19 +12,19 @@ If you believe you have found a security vulnerability in any Microsoft-owned re
 
```

Comment:
Make sure this file is not getting formatted. If you copied the prettierignore it should be there. I think otherwise you'll keep getting an automated PR to revert it to the official version.

Reply:
It's in the prettierignore, I have `/SECURITY.md` in there